docs/source/faq.md

```diff
@@ -38,7 +38,7 @@ The UCX framework is maintained and supported by hardware vendors in addition to
 The framework design, data structures, and components are design to provide highly optimized access to the network hardware. 
 
 * **High level API for a broad range HPC programming models.**  
-UCX provides a high level API implemented in software 'UCP' to fill in the gaps across interconnects. This allows to use a single set of APIs in a library  to implement multiple interconnects. This reduces the level of complexities when implementing libraries such as Open MPI or OpenSHMEM.  Because of this, UCX performance portable  because a single implementation (in Open MPI or OpenSHMEM) will work efficiently on multiple interconnects. (e.g. uGNI, Verbs, libfabrics, etc). 
+UCX provides a high level API implemented in software 'UCP' to fill in the gaps across interconnects. This allows to use a single set of APIs in a library  to implement multiple interconnects. This reduces the level of complexities when implementing libraries such as Open MPI or OpenSHMEM.  Because of this, UCX performance portable  because a single implementation (in Open MPI or OpenSHMEM) will work efficiently on multiple interconnects. (e.g. uGNI, Verbs, etc).
```

Comment:
UCX provides a high level API implemented in software (UCP) to fill in the gaps across interconnects. This allows the use of a single set of APIs in a library to implement multiple interconnects, and reduces the level of complexity when implementing libraries such as Open MPI or OpenSHMEM.  UCX is therefore performance portable because a single implementation (in Open MPI or OpenSHMEM) will work efficiently on multiple interconnects. (e.g. uGNI, Verbs).